todos/constants.js

```diff
@@ -1 +1,14 @@
 export const NAME = 'todos'
+
+// Action Types
```

Comment:
I'd vote to namespace the constants within the file. Instead of exporting each constant separately, we could do:
```
export const actionTypes = {
  ADD: 'todos/add',
  DELETE: 'todos/delete',
  ...
}
```
Same thing for filters.